src/ripple/overlay/tests/manifest_test.cpp

```diff
@@ -179,6 +179,26 @@ class manifest_test : public ripple::TestSuite
                                    boost::filesystem::path (dbName));
     }
 
+    void testGetSignature()
+    {
+        testcase ("getSignature");
+        auto const sk = randomSecretKey();
+        auto const pk = derivePublicKey(KeyType::ed25519, sk);
+        auto const kp = randomKeyPair(KeyType::secp256k1);
+        auto const m = make_Manifest (KeyType::ed25519, sk, kp.first, 0);
+
+        STObject st(sfGeneric);
+        set(st, sfSequence, 0);
+        set(st, sfPublicKey, pk);
+        set(st, sfSigningPubKey, kp.first);
+        Serializer ss;
+        ss.add32(HashPrefix::manifest);
+        st.addWithoutSigningFields(ss);
+        auto const sig = sign(KeyType::ed25519, sk, ss.slice());
+
+        expect (strHex(sig) == strHex(m.getSignature()));
```

Comment:
Missing `#include <ripple/basics/StringUtilities.h>` needed for `strHex` (or the nonunity build fails)